public/planning/poste/fonctions.php

```diff
@@ -62,6 +62,12 @@ function cellule_poste($date, $debut, $fin, $colspan, $output, $poste, $site)
                     $class_tmp[]="striped";
                 }
 
+                if ($elem['depart'] > '0000-00-00' && $elem['depart'] <= $date) {
```

Comment:
On the departure date, the agent is still working

Change this line for 
<pre>if ($elem['depart'] > '0000-00-00' && $elem['depart'] < $date) {</pre>

Reply:
Solved here 7daf8ce5ce069d00033cea6ad2439bbcfe862ec6